roles/lib_openshift/src/lib/base.py

```diff
@@ -9,6 +9,29 @@ class OpenShiftCLIError(Exception):
     pass
 
 
+def locate_oc_binary():
+    ''' Find and return oc binary file '''
+    # https://github.com/openshift/openshift-ansible/issues/3410
+    # oc can be in /usr/local/bin in some cases, but that may not
+    # be in $PATH due to ansible/sudo
+    paths = os.environ.get("PATH", os.defpath).split(os.pathsep) + ['/usr/local/bin', os.path.expanduser('~/bin')]
```

Comment:
So, I would expect that it would use the user's $HOME if not running under sudo, and root's $HOME if running under sudo. I know that is the default behavior outside of Ansible, but I haven't validated it with Ansible.